docs/cudf/source/developer_guide/index.md

```diff
@@ -21,6 +21,7 @@ Additionally, it includes longer sections on more specific topics like testing a
 :maxdepth: 2
 
 library_design
+contributing_guide
```

Comment:
@vyasr I assume you did not expose the contributing guide PR to because it's incomplete. It should be ready to publish after we merged this PR.

Reply:
Yup, thanks